Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -119,9 +162,56 @@ private Vector3 WorldSpacePressDirection
                 return transform.forward;
             }
         }
+        private Transform PushSpaceSourceTransform
+        {
+            get { return movingButtonVisuals != null ? movingButtonVisuals.transform : transform; }
+        }
+
+        private float LocalToWorldScale
+        {
+            get
+            {
+                return transform.TransformVector(WorldSpacePressDirection).magnitude;
+            }
+        }
+        private Vector3 LocalForward
+        {
+            get
```

Comment:
Interesting that LocalForward is defined as the NearInteractionTouchable instead of z in local space.

Reply:
I took this over from how this was determined before - we could also just use Vector3.forward instead because that's what nearinteractiontouchable.localforward points to anyways - i'm not sure what the original intent behind this was. I don't want to dig up too many holes at once

Reply:
actually nevermind - we don't need this at all anymore . i remove it!